index.js

```diff
@@ -23,7 +23,14 @@ const CLIENT_OPTIONS = {
 
 console.log(readFileSync('bigtitle.txt', 'utf8').toString())
 
+const debug = Boolean(process.env.DEBUG)
```

Comment:
```suggestion
const debug = process.env.DEBUG === 'true'
```


![image](https://user-images.githubusercontent.com/22124518/97066231-20821c00-158a-11eb-9a9d-b15817b6898e.png)

Reply:
WTF, I forgot that DotEnv don't transform booleans